banyand/measure/aggregate/aggregate_function.go

```diff
@@ -26,20 +26,20 @@ import (
 )
 
 // Void type contains nothing. It works as a placeholder for type parameters of `Arguments`.
-type Void struct{}
+// It's implemented as int64, but it won't be used as an int64.
+type Void int64
 
-// Input covers possible types of Function's arguments. It synchronizes with `FieldType` in schema.
+// Input covers possible types of Function's arguments. It synchronizes with `FieldType`.
+// It also covers Void type.
 type Input interface {
-	Void | ~int64 | ~float64
+	~int64 | ~float64
```

Comment:
Do you need to separate 'Input' and 'Output'?

Reply:
Laterly we may introduce `HashMap` to Output.